src/Analysis/Engine/Test/LanguageServerTests.cs

```diff
@@ -992,6 +992,19 @@ class C:
             );
         }
 
```

Comment:
This file is getting huge. Can we create LanguageServer folder and then separate FormattingTests file?

Reply:
Would you like me to do that in this PR, or a subsequent one? It might be more straightforward to make a PR to move all of the tests at once, since a lot of the "CreateServer" logic is shared between the tests.